<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1027</width>
    <height>778</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" columnstretch="2,5">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTableWidget" name="tableWidget_Strategy">
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>N</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>С т р а т е г и и</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
    <item row="2" column="0">
     <widget class="QTableWidget" name="tableWidget_Frames">
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>N</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ф р е й м ы</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QTableWidget" name="tableWidget_SlotsByFrame">
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>N</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>С л о т ы  а к т и в н о г о  Ф р е й м а</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QTableWidget" name="tableWidget_Products">
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>32</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>N</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Продукции: У с л о в и я  р е а л и з а ц и и</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>З а к л ю ч е н и е</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1027</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="action_4"/>
    <addaction name="action_3"/>
    <addaction name="separator"/>
    <addaction name="actionRecent"/>
    <addaction name="separator"/>
    <addaction name="action_5"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Объекты</string>
    </property>
    <widget class="QMenu" name="menu_5">
     <property name="title">
      <string>Стратегии</string>
     </property>
     <addaction name="action_17"/>
     <addaction name="action_18"/>
     <addaction name="separator"/>
     <addaction name="action_20"/>
     <addaction name="separator"/>
     <addaction name="action_22"/>
     <addaction name="action_23"/>
    </widget>
    <widget class="QMenu" name="menu_6">
     <property name="title">
      <string>Продукции</string>
     </property>
     <addaction name="action_24"/>
     <addaction name="action_25"/>
     <addaction name="action_26"/>
     <addaction name="action_27"/>
    </widget>
    <widget class="QMenu" name="menu_7">
     <property name="title">
      <string>Фреймы</string>
     </property>
     <addaction name="action_28"/>
     <addaction name="action_29"/>
     <addaction name="separator"/>
     <addaction name="action_31"/>
    </widget>
    <widget class="QMenu" name="menu_8">
     <property name="title">
      <string>Слоты</string>
     </property>
     <addaction name="action_32"/>
     <addaction name="action_33"/>
     <addaction name="action_34"/>
    </widget>
    <addaction name="menu_5"/>
    <addaction name="menu_6"/>
    <addaction name="separator"/>
    <addaction name="menu_7"/>
    <addaction name="menu_8"/>
    <addaction name="separator"/>
    <addaction name="action_12"/>
    <addaction name="action_modules"/>
    <addaction name="action_14"/>
    <addaction name="separator"/>
    <addaction name="action_16"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Просмотр</string>
    </property>
    <addaction name="action_37"/>
    <addaction name="action_38"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="action_39"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>Новый</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Открыть...</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Сохранить как...</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="actionRecent">
   <property name="text">
    <string>Recent File</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_12">
   <property name="text">
    <string>Атрибуты...</string>
   </property>
  </action>
  <action name="action_modules">
   <property name="text">
    <string>Модули...</string>
   </property>
  </action>
  <action name="action_14">
   <property name="text">
    <string>Меню...</string>
   </property>
  </action>
  <action name="action_16">
   <property name="text">
    <string>Хранилище значений...</string>
   </property>
  </action>
  <action name="action_17">
   <property name="text">
    <string>Новая</string>
   </property>
  </action>
  <action name="action_18">
   <property name="text">
    <string>Удалить</string>
   </property>
  </action>
  <action name="action_20">
   <property name="text">
    <string>Комментарий</string>
   </property>
  </action>
  <action name="action_22">
   <property name="text">
    <string>Экспорт</string>
   </property>
  </action>
  <action name="action_23">
   <property name="text">
    <string>Импорт</string>
   </property>
  </action>
  <action name="action_24">
   <property name="text">
    <string>Новая</string>
   </property>
  </action>
  <action name="action_25">
   <property name="text">
    <string>Редактировать</string>
   </property>
  </action>
  <action name="action_26">
   <property name="text">
    <string>Удалить</string>
   </property>
  </action>
  <action name="action_27">
   <property name="text">
    <string>Переместить</string>
   </property>
  </action>
  <action name="action_28">
   <property name="text">
    <string>Новый</string>
   </property>
  </action>
  <action name="action_29">
   <property name="text">
    <string>Удалить</string>
   </property>
  </action>
  <action name="action_31">
   <property name="text">
    <string>Комментарий</string>
   </property>
  </action>
  <action name="action_32">
   <property name="text">
    <string>Новый</string>
   </property>
  </action>
  <action name="action_33">
   <property name="text">
    <string>Редактировать</string>
   </property>
  </action>
  <action name="action_34">
   <property name="text">
    <string>Удалить</string>
   </property>
  </action>
  <action name="action_37">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Линейка</string>
   </property>
  </action>
  <action name="action_38">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Состояние</string>
   </property>
  </action>
  <action name="action_39">
   <property name="text">
    <string>О программе...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
